connection_test.go

```diff
@@ -284,7 +284,7 @@ func TestBadRequest(t *testing.T) {
 		assert.Equal(t, ErrCodeBadRequest, GetSystemErrorCode(err))
 
 		calls := relay.NewMockStats()
-		calls.Add(ts.ServiceName(), ts.ServiceName(), "Noone").Failed("bad-request").End()
+		calls.Add(ts.ServiceName(), ts.ServiceName(), "Noone").Fluent().Failed("bad-request").End()
```

Comment:
`calls.Add` can't return the fluent version, because `relay.MockStats` needs to satisfy the `relay.CallStats` interface.

I don't mind adding a separate method to return the fluent version, but it seems less flexible without much benefit - is `AddFluent(...)` any better than `Add(...).Fluent()`?